src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -52,4 +52,16 @@
 
   /** Name of historical data queue time tracker file 2. */
   static final String QUEUE_TIME_FILE_2_NAME = "histDataTime2";
+
+  /** Minimum amount of Historical FIFO queue that can be set by library users. */
+  static final long MIN_QUEUE_SPAN_MINS = 1;
+
+  /** Default maximum amount of time Historical FIFO can get behind. 48hours in ms. */
```

Comment:
```suggestion
  /** Default maximum amount of time Historical FIFO can get behind. 48 hours in ms. */
```

or
```suggestion
  /** Default maximum amount of time Historical FIFO can get behind. */
```

Reply:
I generally like to leave the value out of comments. If someone changes the value there is a chance they may not update the comment, then its a code maintenance issue. If someone really wanted to know what the value was they could just use a calculator.
![image](https://user-images.githubusercontent.com/17338468/152413632-2f664167-0075-4f55-8d82-87827eaa1df2.png).

Another way to do it and have it documented in the code is 
`
long MAX_HIST_FIFO_GET_BEHIND_HOURS = 48L;
long MAX_HIST_FIFO_GET_BEHIND_MS = MAX_HIST_FIFO_GET_BEHIND_HOURS * HOURS_PER_DAY * MINUTES_PER_HOUR * SECONDS_PER_MINUTE * MILLIS_PER_SECOND;
`

But thats a bit nasty looking

OR cleaner would be 
`
long MAX_HIST_FIFO_GET_BEHIND_HOURS = 48L;
long MAX_HIST_FIFO_GET_BEHIND_MS = MAX_HIST_FIFO_GET_BEHIND_HOURS * MILLISECONDS_PER_DAY;
`

Reply:
wait, somethings not right here. let me look at this

Reply:
@it-hms I think you have 48 days in milliseconds here not 48 hours.

Reply:
This can be simplified further using our TimeUnit class.

For example,

```java
    int millisInOneHour = SCTimeUnit.HOURS.toMillis(1);
    int millisInFourHours = SCTimeUnit.HOURS.toMillis(4);
```

Reply:
Thanks for checking this ... resolved.